src/com/lilithsthrone/utils/Units.java

```diff
@@ -574,7 +574,7 @@ public static String getMonthName(int month) {
      */
     public static float round(double value, int places) {
         if (places < 0) throw new IllegalArgumentException("Amount of fractional places cannot be less than 0.");
-        return BigDecimal.valueOf(value).setScale(places, BigDecimal.ROUND_HALF_UP).floatValue();
+        return BigDecimal.valueOf(value).setScale(places, java.math.RoundingMode.HALF_UP).floatValue();
```

Comment:
RoundingMode is already on the import list, so replace with RoundingMode.HALF_UP (without the java.math. prefix)

Reply:
Applied the suggested change